mysql/update_statement.go

```diff
@@ -12,15 +12,19 @@ type UpdateStatement interface {
 	MODEL(data interface{}) UpdateStatement
 
 	WHERE(expression BoolExpression) UpdateStatement
+	LIMIT(limit int64) UpdateStatement
+	INNER_JOIN(table ReadableTable, onCondition BoolExpression) UpdateStatement
```

Comment:
Multi table update is already supported, check this test: https://github.com/go-jet/jet/blob/00b8155f74dbeb51f5f5c7351f002c632a22f401/tests/mysql/update_dvds_test.go#L11-L15

Reply:
Updated this.